src/StoreApi/Schemas/ProductCategorySchema.php

```diff
@@ -102,13 +102,18 @@ public function get_item_response( $term ) {
 	protected function get_category_review_count( $term ) {
 		global $wpdb;
 
+		$term_children_ids = get_term_children( $term->term_id, 'product_cat' );
```

Comment:
Might need error handling here. See https://github.com/woocommerce/woocommerce/blob/a5333ffd1f15c22d5298d759c6b5fb166d087306/includes/wc-term-functions.php#L475-L482

Reply:
Ah, thanks for catching! I adapted the code to look like the example you posted.